.github/workflows/ci.yml

```
@@ -0,0 +1,252 @@
+      - name: Install Playwright browsers
+        if: matrix.node-version == 20
+        run: pnpm --filter @bluelight-hub/frontend exec playwright install --with-deps chromium
+
+      - name: Run E2E tests
+        if: matrix.node-version == 20
+        env:
+          CI: true
+        run: pnpm --filter @bluelight-hub/frontend test:e2e:auth
+
+      # Upload test artifacts on failure
+      - uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)